README.md

```diff
@@ -1 +1,11 @@
-# hw01
\ No newline at end of file
+# __*Homework 1*__
+* __Name__: Moksha Sharma
```

Comment:
Leave a blank line after headers. This prevents rendering issues from occurring (it will be more prevalent once you combine markdown with R code)